### ISSUE TITLE
Unable to run locally

### ISSUE THREAD

**Issue by rptabo** [apollostack/mock-demo#1]
I was really excited to run this locally so I could have a starting point for my own mock. However, after several attempts, I could not get this to work and ran into several issues.

First, `npm install` produced the following warnings:

```
npm WARN graphiql@0.6.6 requires a peer of graphql@^0.4.16 but none was installed.
npm WARN codemirror-graphql@0.3.1 requires a peer of graphql@^0.4.16 but none was installed.
```

...and here's a snippet from the install:

```
├─┬ graphiql@0.6.6
│ ├── codemirror@5.14.2
│ ├── codemirror-graphql@0.3.1
│ ├── UNMET PEER DEPENDENCY graphql@^0.4.16
│ └── marked@0.3.5
├── UNMET PEER DEPENDENCY graphql@^0.4.16
```

I tried to fix this by installing those versions directly, both from the command line, and also adding as both a dependency and then as a devDependency in package.json. However, once that version was installed, there was the following output:

```
mock-demo@1.0.0 /Users/rpo/[snip]/mock-demo
└── graphql@0.4.18  extraneous

npm WARN express-graphql@0.5.1 requires a peer of graphql@^0.5.0 but none was installed.
```

I could not figure out how to install both 0.4.16 and 0.5.0 versions of graphql.

No matter what I tried (fresh install ignoring warnings, installing version 0.4.16 and not worrying about 0.5.0, and vice versa), whenever I ran `npm start`, I got the output in the attached file. Here are some highlights:

```
[...]
  [825] ./~/statuses/index.js 1.14 kB {0} [built] [1 error]
[...]
  [842] ./~/iconv-lite/encodings/dbcs-data.js 8.11 kB {0} [built] [1 error]
[...]

ERROR in ./~/statuses/codes.json
Module parse failed: /Users/rpo/[snip]/mock-demo/node_modules/statuses/codes.json Unexpected token (2:7)
You may need an appropriate loader to handle this file type.
SyntaxError: Unexpected token (2:7)
    at Parser.pp.raise (/Users/rpo/[snip]/mock-demo/node_modules/webpack/node_modules/acorn/dist/acorn.js:920:13)
[...]

ERROR in ./~/iconv-lite/encodings/tables/gb18030-ranges.json
Module parse failed: /Users/rpo/[snip]/mock-demo/node_modules/iconv-lite/encodings/tables/gb18030-ranges.json Unexpected token (1:9)
You may need an appropriate loader to handle this file type.
SyntaxError: Unexpected token (1:9)
    at Parser.pp.raise (/Users/rpo/[snip]/mock-demo/node_modules/webpack/node_modules/acorn/dist/acorn.js:920:13)
```

I just see a blank page at http://localhost:8080. It is serving index.html in the root directory (with `<script type="text/javascript" src="bundle.js">`), but nothing is being displayed like it as at https://apollostack.github.io/mock-demo

I am running Mac OS X El Capitan 10.11.4 with node v6.1.0 and npm v3.8.6 running in zsh.

I did some searching and couldn't find a solution to this issue. I must admit I'm new to the world of React/GraphQL and its supporting technologies such as webpack. I thought this mock demo would be a perfect entry point, so I hope to get this running.

Thanks in advance for any troubleshooting suggestions.

[mock-demo-log.txt](https://github.com/apollostack/mock-demo/files/264366/mock-demo-log.txt)

**Comment by helfer** [comment]
Thanks for reporting this @rptabo, looks like some dependency updated and broke the build because it included json files (the peerDependency thing wasn't actually what stopped you from bundling, but I fixed it anyway). For good measure I've shrinkwrapped the versions.

However, unfortunately `npm install` may hang with shrinkwrap https://github.com/npm/npm/issues/10579. If that happens to you, just remove the npm-shinkrwrap.json file and try again. Let me know if it works!

Also, the demo apps on [docs.apollostack.com](http://docs.apollostack.com) may be more informative than this mock-demo, which was built as a single static page.
